packs/redis/config.yml

```diff
@@ -5,8 +5,26 @@ name: Redis
 
 # Description of the pack
 description: |+
-  Official New Relic pack for Redis
-  Use this pack together with the New Relic Redis On Host Integration to get insight into the performance of your Redis instances.
+  # A Complete Redis Monitoring System
+  ## Monitoring Redis
+  Redis is one of the most performant databases available to developers. It operates in-memory and consequently achieves I/O times orders of magnitude faster than traditional database systems. It can be used as a lightweight, distributed queue and also integrates useful features such as a pub/sub notification system.
+
+  Redis is popular in the gaming, IoT, and mobile industries due to its easy onboarding and blazing-fast performance. It includes several data structures which make it ready to use right out of the box.
+
+  New Relic provides a Redis dashboard which allows you to monitor your Redis instances.
+
+  ## New Relic - A Perfect Tool to Monitor Redis
+  Redis is known for its speed, so ensuring that it stays operating at peak performance is paramount. Slowdowns can lead to a compromised user experience or even a complete application failure.
+
+  Monitoring Redis is a passive process, but the New Relic tool provides actionable insights into the health of a Redis system. In short, New Relic is more than just a monitor. It supports custom charts and custom queries using data from New Relic Insights, and pre-built dashboards are available for those who don’t need custom configurations. All critical performance and health metrics are monitored.
+
+  ## New Relic Redis Kit Features
```

Comment:
I would advise against the use of 'Kit' and always quickstart to keep it consistent
@bradleycamacho what do you think?

Reply:
We should always use quickstart, kit is a slang term IMHO being used occasionally in conversation, we can try to stop calling them kits internally as well.